R/utils.R

```diff
@@ -465,3 +465,21 @@ infer_sectors_and_technologies <- function(baseline_scenario, shock_scenario, sc
 
   return(list(sectors = shared_sectors, technologies = technologies))
 }
+
+
+infer_scenario_type <- function(baseline_scenario, shock_scenario) {
+  if (grepl("NGFS2021", baseline_scenario) & grepl("NGFS2021", shock_scenario)) {
```

Comment:
nicer: if (condA) {bla} ifelse(condB){bla} else {bla}

You dont have to put any content in the else condition then.